cmd/fleetctl/fleetctl/preview.go

```diff
@@ -144,11 +142,6 @@ Use the stop and reset subcommands to manage the server and dependencies once st
 				Usage: "Use custom update TUF root keys",
 				Value: "",
 			},
-			&cli.StringFlag{
-				Name:  stdQueryLibFilePath,
-				Usage: "Use custom standard query library yml file (used for development/testing)",
-				Value: "",
-			},
```

Comment:
We need something similar to test starter library yaml changes on the PRs (because once merged main it's "released")

Reply:
Good catch, thanks. I've added back a new flag to for the starter library.